lib/cli/run-helpers.js

```diff
@@ -295,7 +198,7 @@ exports.runMocha = (
   files = []
 ) => {
   if (watch) {
-    exports.watchRun(mocha, {extension, grep, ui, files});
+    watchRun(mocha, {extension, grep, ui, files, cwd});
```

Comment:
Shouldn't this still be exported here?

Reply:
`lib/cli/run-helpers` is marked as a private module and since the export was never used I removed it. If you feel that we should keep it for backwards compatibility reasons then I’d be ok adding it back.

Reply:
Prefer to keep the `export`.

Reply:
Done